lib/routes/routeUtapi.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
+const http = require('http');
+const { errors } = require('arsenal');
+const { responseJSONBody } = require('arsenal').s3routes.routesUtils;
+const { config } = require('../Config');
+
+function _decodeURI(uri) {
+    // do the same decoding as in S3 server
+    return decodeURIComponent(uri.replace(/\+/g, ' '));
+}
+
+function _createUtapiRequestOptions(req) {
+    const parsedUrl = url.parse(req.url, true);
+    const reqPath = _decodeURI(parsedUrl.pathname);
+    // path will begin with /_/utapi
+    const utapiPath = `${reqPath.substring(8)}${parsedUrl.search}`;
+    const utapiPort = config.utapi.port || 8100;
```

Comment:
Same thing here, please check if `config.utapi` is defined before accessing the port/host property.